carapace-server/src/main/java/org/carapaceproxy/server/mapper/StandardEndpointMapper.java

```diff
@@ -343,6 +385,18 @@ public MapResult map(HttpRequest request, String userId, String sessionId, Backe
         }
     }
 
+    private HttpHeaders headersForAction(ActionConfiguration action) {
+        HttpHeaders _headers = new DefaultHttpHeaders();
```

Comment:
are we always creating a new object ?
is it really needed ?
we are on the 'hot path' we should save as much resources as possible